doc/api/modules.md

```diff
@@ -288,8 +294,13 @@ The core modules are defined within the Node.js source and are located in the
 `lib/` folder.
 
 Core modules are always preferentially loaded if their identifier is
-passed to `require()`. For instance, `require('http')` will always
-return the built in HTTP module, even if there is a file by that name.
+passed to `require()`. For instance, `require('node:http')` will always
+return the built in HTTP module, even if there is a file or `require.cache`
+entry by that name.
```

Comment:
Correct me if I'm wrong, but there's no possibility to have a `node:fs` module today, the part where it says `even if there's a file by that name` doesn't apply. Also, the information regarding non-prefixed `require` calls still holds, and shouldn't be removed.

```suggestion
passed to `require()`. For instance, `require('http')` will always
return the built in HTTP module, even if there is a file by that name.

Core modules can also be identify using `node:` prefix, in which case it bypasses the `require` cache. For instance, `require('node:http')` will always return the built in HTTP module, even if there is `require.cache` entry by that name.
```

Reply:
> EDIT: I was wrong, you could have a `node:fs` package today, and this would technically be a breaking change for a theoretical user that use that for one of their packages.

Note that that doesn’t apply to the **NTFS** file system, and thus **Windows**, since U+003A (`:`) is a reserved character on **NTFS** used to address alternate data streams.